src/components/subsetDraft/SubsetMetadata.jsx

```diff
@@ -1,95 +1,152 @@
+        subset.dispatch({ action: "description_add" });
+    const { ssbsections, classificationfamilies } = useContext(AppContext);
     const { t } = useTranslation();
 
+    const { isShowing, dismiss, submit, handleDateChange } = useChangeDateModal();
+    
     return (
         <>
-            <Title size={3}>{t('Metadata')}</Title>
-            <TextLanguageFieldset title={t('Names')} items={subset.draft.name}
-                                  add={() => subset.dispatch({action: 'name_add'})}
-                                  remove={(index) => subset.dispatch({action: 'name_remove', data: index})}
-                                  handle={() => subset.dispatch({action: 'update'})}
-                                  size={{cols: 65, rows: 1}}
-                                  prefix={subsetDraft.namePrefix}
+            <Title size={3}>{t("Metadata")}</Title>
+            <TextLanguageFieldset
+                title={t("Names")}
+                items={subset.draft.name}
+                add={() => subset.dispatch({ action: "name_add" })}
+                remove={index =>
+                    subset.dispatch({ action: "name_remove", data: index })
+                }
+                handle={() => subset.dispatch({ action: "update" })}
+                size={{ cols: 65, rows: 1 }}
+                prefix={subsetDraft.namePrefix}
             />
 
-            <section style={{margin: '5px 0 5px 0'}}>
-                <div style={{float: 'left', marginRight: '20px', padding: '0'}}>
-                    <label style={{display: 'block', fontSize: '16px', fontFamily: 'Roboto'}}
-                           htmlFor="from_date">{t('Valid from')}: </label>
-                    <DatePicker id='from_date' style={{display: 'block'}}
-                                value={subset.draft.validFrom}
-                                onChange={(date) => subset.dispatch({action: 'from', data: date})}
-                                clearIcon={null}
-                                format='dd.MM.y'
-                                locale={languages.find(i => i.default).IETF}
-                                className='datepicker'/>
+            <section style={{ margin: "5px 0 5px 0" }}>
+                <div
+                    style={{ float: "left", marginRight: "20px", padding: "0" }}
+                >
+                    <label
+                        style={{
+                            display: "block",
+                            fontSize: "16px",
+                            fontFamily: "Roboto"
+                        }}
+                        htmlFor='from_date'
+                    >
+                        {t("Valid from")}:{" "}
+                    </label>
+                    <DatePicker
+                        id='from_date'
+                        style={{ display: "block" }}
+                        value={subset.draft.validFrom}
+                        onChange={date => handleDateChange(date, 'from')}
```

Comment:
Here the component does not know if the AppContext want to show Modal or can throw validation error. 
The only thing it knows, is the date the user has chosen. It informs the AppContext about this action by calling action `subset.dispatch({action "from", date})`.